src/rs_quark_assets/components/A2HSBanner.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div v-if="eventTriggered !== 'triggered'">
```

Comment:
```suggestion
  <div v-if="eventTriggered !== 'triggered'">
```

nitpick: I believe you could use a `<template>` here in stead to avoid unnecessary DOM